ruby/server.rb

```diff
@@ -30,3 +31,26 @@
   end
   redirect '/'
 end
+
+post '/organize' do
+  @wishlist_for_user = []
+  if params["name"]
```

Comment:
the style guide suggests using single quotes for ruby code, unless you're doing string interpolation